README.md

```diff
@@ -0,0 +1,60 @@
+## vmware-govcd
```

Comment:
Please fix the readme with appropriate instructions, merging the content of the original README and the new one coming from the other side of the merge